sendgrid-client/build.gradle

```diff
@@ -57,6 +57,5 @@ dependencies {
   testImplementation libraries.junit5
   testImplementation group: 'org.springframework.boot', name: 'spring-boot-starter-test', {
     exclude group: 'junit', module: 'junit'
```

Comment:
I think this is also excluded already, please verify.

Reply:
yes, removed